### ISSUE TITLE
AOT fails on angular@5 angulal/cli@1.7

### ISSUE THREAD

**Issue by amitport** [DesertFoxNV/ngx-autosize-input#5]
after updating from `ngx-autosize-input@1.0.3` to `ngx-autosize-input@2.0.2`

without AOT; you get a warning:
```
WARNING in ./node_modules/ngx-autosize-input/node_modules/@angular/core/src/linker/system_js_ng_module_factory_loader.js
71:15-36 Critical dependency: the request of a dependency is an expression
```

with AOT; a failure:
```
ERROR in Error during template compile of 'NgModule'
  Function calls are not supported in decorators but 'makeDecorator' was called.
: Unexpected value 'autoSizeInputModule in <my-project-path>/node_modules/ngx-autosize-input/autoSizeInput.module.d.ts' imported by the module 'AssetsModule in <my-project-path>/src/root/app/ass
ets/asset.module.ts'. Please add a @NgModule annotation.
Error during template compile of 'Directive'
  Function calls are not supported in decorators but 'makeDecorator' was called.

```

**Comment by joshuawwright** [comment]
@amitport What version of angular 5 are you using?

**Comment by amitport** [comment]
@DesertFoxNV 5.2.7

**Comment by joshuawwright** [comment]
@amitport 
OK, the bug should be fixed now in version 2.1. Note that I updated the ReadMe file. I have changed the directive name from "appAutoSizeInput" to "AutoSizeInput"
